otx/cli/utils/multi_gpu.py

```diff
@@ -191,7 +191,7 @@ def setup_multi_gpu_train(
     def finalize(self):
         """Join all child processes."""
         for p in self._processes:
-            if p.join(10) is None and p.exitcode is None:
+            if p.join(30) is None and p.exitcode is None:
```

Comment:
@eunwoosh , it seems to bit hard-coded. Could we avoid ..? (Not essential)

Reply:
Because when child processes are done isn't certain, it's inevitable, I think.